x-pack/test/functional/apps/machine_learning/anomaly_detection/advanced_job.ts

```diff
@@ -279,6 +279,7 @@ export default function({ getService }: FtrProviderContext) {
     before(async () => {
       await esArchiver.load('ml/ecommerce');
       await ml.api.createCalendar('wizard-test-calendar');
+      await ml.securityUI.loginAs('ml_poweruser');
```

Comment:
this code is repeated a lot, would it be worth having wrapper functions for commonly used users?
something like `await ml.securityUI.loginAsPowerUser();`
or possibly having the usernames in a constant like an enum? the string `ml_poweruser` is repeated 31 times in this PR.
I'm thinking from the point of view of writing a new test, the only way to know what users are available to me would to look through the `security_common.ts`.

Reply:
Great idea, thanks! Done in def08dd